src/templates/entity/entity.template.ts

```diff
@@ -0,0 +1,124 @@
+import { allOfDereference } from '@templates/lib/allOfdereference'
+import { generateTsFile } from '@utils/generateTsFile'
+import { getFileImports } from '@utils/getFileImports'
+import * as _ from 'lodash'
+
+class EntityFileFactory {
+    private enums: Record<string, any> = {}
+
+    private imports: Record<string, Set<string>> = {
+        ['typeorm']: new Set(),
+        ['@nestjs/swagger']: new Set(),
+    }
+
+    private rootPath: string
+
+    private serviceName: string
+
+    private entities: { title: string; code: string }[]
+
+    constructor(rootPath: string, serviceName: string, entitySchemas: [string, any][]) {
+        this.rootPath = rootPath
+        this.serviceName = serviceName
+        this.imports['typeorm'].add('Column')
+        this.imports['typeorm'].add('Entity')
+        this.imports['typeorm'].add('PrimaryGeneratedColumn')
+        this.imports['@nestjs/swagger'].add('OmitType')
+
+        this.entities = entitySchemas.map(([_title, _data]) => {
+            const obj = {
+                title: _title,
+                code: this.createEntityClass(_title, _data),
+            }
+            return obj
+        })
```

Comment:
```suggestion
        this.entities = entitySchemas.map(([title, data]) => ({
                title,
                code: this.createEntityClass(title, data),
            })
        })
```

не понимаю зачем здесь underscore. В данном случае он блокирует упрощенную запись значения title.
+ obj можно не создавать и упростить запись еще больше

Reply:
done